src/storage/hummock_test/src/test_utils.rs

```diff
@@ -353,3 +356,86 @@ pub async fn register_test_tables<S: MetaStore>(
     )
     .await;
 }
+
+pub async fn register_test_tables_with_catalog<S: MetaStore>(
+    filter_key_extractor_manager: &FilterKeyExtractorManagerRef,
+    hummock_manager_ref: &HummockManagerRef<S>,
+    tables: &[ProstTable],
+) {
+    update_filter_key_extractor_for_tables(filter_key_extractor_manager, tables);
+    let table_ids = tables.iter().map(|t| t.id).collect_vec();
+    register_table_ids_to_compaction_group(
+        hummock_manager_ref,
+        &table_ids,
+        StaticCompactionGroupId::StateDefault.into(),
+    )
+    .await;
+}
+
+#[allow(dead_code)]
+pub struct HummockTestEnv {
+    pub storage: HummockStorage,
+    pub manager: HummockManagerRef<MemStore>,
+    pub meta_client: Arc<MockHummockMetaClient>,
+}
+
+impl HummockTestEnv {
+    pub async fn register_table_id(&self, table_id: TableId) {
+        register_test_tables_with_id(
+            self.storage.filter_key_extractor_manager(),
+            &self.manager,
+            &[table_id.table_id()],
+        )
+        .await;
+    }
+
+    pub async fn register_table(&self, table: ProstTable) {
+        register_test_tables_with_catalog(
+            self.storage.filter_key_extractor_manager(),
+            &self.manager,
+            &[table],
+        )
+        .await;
+    }
+
+    // Seal, sync and commit a epoch.
+    // On completion of this function call, the provided epoch should be committed and visible.
+    pub async fn commit_epoch(&self, epoch: u64) {
+        let res = self.storage.seal_and_sync_epoch(epoch).await.unwrap();
+        self.meta_client
+            .commit_epoch(epoch, res.uncommitted_ssts)
+            .await
+            .unwrap();
+        self.storage.try_wait_epoch_for_test(epoch).await;
+    }
+}
+
+pub async fn prepare_hummock_test_env() -> HummockTestEnv {
+    let sstable_store = mock_sstable_store();
+    let hummock_options = Arc::new(default_config_for_test());
+    let (env, hummock_manager_ref, _cluster_manager_ref, worker_node) =
+        setup_compute_env(8080).await;
+
+    let hummock_meta_client = Arc::new(MockHummockMetaClient::new(
+        hummock_manager_ref.clone(),
+        worker_node.id,
+    ));
+
+    let notification_client =
+        get_test_notification_client(env, hummock_manager_ref.clone(), worker_node.clone());
```

Comment:
The same question, do we need to name uniformly

for example,
1. default_config_for_test + get_notification_client_for_test
2. get_test_config + get_test_notification_client

Reply:
changed to 1